### PR TITLE
Add comment about the necessity camera parameters

### PR DESCRIPTION
**Pull request by chezzdev**
Checks:

* [ ] Update changelog
* [ ] Rebase to `dev` branch
* [x] Assign reviewers

Linked issues:
Resolves https://github.com/mapbox/mapbox-vision/issues/2337

### CODE REVIEW

File: Examples/VisionExamples/Examples/ExternalCameraViewController.swift [hunk 1]
```diff
@@ -3,7 +3,13 @@ import MapboxVision
 import UIKit
 
 /**
- * "External camera" example demonstrates how to create a custom source of video stream and pass it to `VisionManager`.
+ * "External camera" example demonstrates how to create a custom source of a video stream
+ * and pass it to `VisionManager`.
+ * **Important:** To enable most of the features of Vision SDK like world description, AR and Safety, the object
```
Comment (dersim-davaod) [inline Examples/VisionExamples/Examples/ExternalCameraViewController.swift:8]: nit: 
```suggestion
 * **Important:** To enable most of the features of the Vision SDK like world description, AR and Safety, the object
```

File: Examples/VisionExamples/Examples/ExternalCameraViewController.swift [hunk 2]
```diff
@@ -54,6 +60,11 @@ class FileVideoSource: ObservableVideoSource {
                     // construct `VideoSample` specifying the format of image contained in a sample buffer
                     let videoSample = VideoSample(buffer: buffer, format: .BGRA)
                     observer.videoSource(self, didOutput: videoSample)
+
+                    // Obtain camera parameters and set them each time they change
```
Comment (dersim-davaod) [inline Examples/VisionExamples/Examples/ExternalCameraViewController.swift:64]: nit: 
```suggestion
                    // Uncomment code below to obtain camera parameters and set them each time they change
```
?

File: Examples/VisionExamples/Examples/ExternalCameraViewController.swift (hunk 1 above)
Comment (chezzdev) [reply Examples/VisionExamples/Examples/ExternalCameraViewController.swift:8]: 👍

File: Examples/VisionExamples/Examples/ExternalCameraViewController.swift (hunk 2 above)
Comment (chezzdev) [reply Examples/VisionExamples/Examples/ExternalCameraViewController.swift:64]: Not really. Just uncommenting won't get you the camera parameters. This is only a demonstration of how they can be constructed and passed.

File: Examples/VisionExamples/Examples/ExternalCameraViewController.swift (hunk 1 above)
Comment (bernardo-martinez) [inline Examples/VisionExamples/Examples/ExternalCameraViewController.swift:8]: when implementing through the internal camera do i need to add CameraParameters too? if yes: how?

**Pull request merged**